components/brave_wallet_ui/components/extension/encryption-key-panel/index.tsx

```diff
@@ -42,18 +43,19 @@ function EncryptionKeyPanel (props: Props) {
     accounts,
     selectedNetwork,
     encryptionKeyPayload,
+    decryptPayload,
     onProvideOrAllow,
     onCancel
   } = props
   const [isDecrypted, setIsDecrypted] = React.useState<boolean>(false)
 
   const foundAccountName = React.useMemo(() => {
-    return accounts.find((account) => account.address.toLowerCase() === encryptionKeyPayload.address.toLowerCase())?.name
-  }, [encryptionKeyPayload])
+    return accounts.find((account) => account.address.toLowerCase() === (encryptionKeyPayload || decryptPayload).address.toLowerCase())?.name
+  }, [encryptionKeyPayload, decryptPayload])
 
   const orb = React.useMemo(() => {
-    return create({ seed: encryptionKeyPayload.address.toLowerCase(), size: 8, scale: 16 }).toDataURL()
-  }, [encryptionKeyPayload])
+    return create({ seed: (encryptionKeyPayload || decryptPayload).address.toLowerCase(), size: 8, scale: 16 }).toDataURL()
```

Comment:
Same here!

Reply:
Fixed in 8999423ff185b6a3e87fa42ac28dfe06e34a6c36